contracts/src/BeefyClient.sol

```diff
@@ -448,22 +448,10 @@ contract BeefyClient {
     ) internal pure returns (uint256) {
         // Start with the minimum number of signatures.
         uint256 numRequiredSignatures = minRequiredSignatures;
-
-        // We must subtract minimumSignatures from the number of validators or we might end up
-        // requiring more signatures than there are validators.
-        uint256 extraValidatorsLen = validatorSetLen.saturatingSub(minRequiredSignatures);
-        if (extraValidatorsLen > 1) {
-            numRequiredSignatures += Math.log2(extraValidatorsLen, Math.Rounding.Ceil);
-        } else if (extraValidatorsLen == 1) {
-            // log2 returns 0 when there is one validator so when this case is hit the single
-            // validator needs to be verified.
-            numRequiredSignatures += 1;
-        }
-
-        if (signatureUsageCount > 0) {
```

Comment:
We do not need to check `signatureUsageCount ` as `Math.log2(signatureUsageCount, Math.Rounding.Ceil)` returns `0` with `0` as input.